test/testdata/container_oom.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
could you do something like I did here https://github.com/kubernetes-incubator/cri-o/blob/master/test/ctr.bats#L586-L588? I'm not liking all this _almost equal_ configurations.